Jenkinsfile

```diff
@@ -804,9 +804,10 @@ def archiveTestOutput(Map args = [:]) {
             cmd(label: "Archive system tests files", script: 'mage packageSystemTests')
           }
         }
-        def fileName = 'build/system-tests-*.tar.gz' // see dev-tools/mage/target/common/package.go#PackageSystemTests method
+        def uploadDirName = 'build'
+        def fileName = uploadDirName + "/system-tests-*.tar.gz" // see dev-tools/mage/target/common/package.go#PackageSystemTests method
         dir("${BASE_DIR}"){
-          cmd(label: "List files to upload", script: "ls -l ${BASE_DIR}/${fileName}")
+          cmd(label: "List files to upload", script: "ls -l ${BASE_DIR}/${uploadDirName}")
```

Comment:
IIRC this step is launched also on windows system, so `ls` could fail (depends on the windows version)

Reply:
Ah good point! I will refactor it to be cross-platform, thanks!

Reply:
there is a util step to list files
https://github.com/jenkinsci/pipeline-utility-steps-plugin/blob/master/src/main/resources/org/jenkinsci/plugins/pipeline/utility/steps/fs/FindFilesStep/help.html

Reply:
I see this step will retrieve an empty array in the case of no entries: https://github.com/jenkinsci/pipeline-utility-steps-plugin/blob/master/src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/fs/FindFilesStepExecution.java#L63

I think we can use it, thanks!